Mage/src/main/java/mage/abilities/costs/common/ExileFromGraveCost.java

```diff
@@ -77,6 +85,9 @@ public boolean pay(Ability ability, Game game, Ability source, UUID controllerId
                 Cards cardsToExile = new CardsImpl();
                 cardsToExile.addAll(exiledCards);
                 controller.moveCards(cardsToExile, Zone.EXILED, ability, game);
+                if (setTargetPointer) {
+                    source.getEffects().setTargetPointer(new FixedTarget(targets.getFirstTarget(), game));
```

Comment:
Current cost support multiple target cards, so the target pointer must be for multiple cards too (`FixedTargets` instead `FixedTarget`).

Reply:
OK.  PR already got merged so I changed this in 5e4b5239d86cf250cf156ad2a68e77494cc08d81